code/controllers/configuration/entries/general.dm

```diff
@@ -205,6 +205,13 @@
 
 /datum/config_entry/flag/use_exp_restrictions_admin_bypass
 
+/datum/config_entry/flag/use_low_living_hour_intern
+
+/datum/config_entry/number/use_low_living_hour_intern_hours
+	config_entry_value = 0
```

Comment:
:-1:, oranges directly requested it default to a different value

Reply:
If it's not set, I use number/use_exp_restrictions_heads_hours as a fallback.

If THAT's not set, I use INTERN_THRESHOLD_FALLBACK_HOURS which is #defined at 15.